laceworksdk/api/policies.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+"""
+Lacework Policies API wrapper.
+"""
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class PoliciesAPI(object):
+
+    def __init__(self, session):
+        """
+        Initializes the PoliciesAPI object.
+
+        :param session: An instance of the HttpSession class
+
+        :return PoliciesAPI object.
+        """
+
+        super(PoliciesAPI, self).__init__()
+
+        self._session = session
+
+    def create(self,
+               policy_type,
+               query_id,
+               enabled,
+               title,
+               description,
+               remediation,
+               severity,
+               alert_enabled,
+               alert_profile,
+               evaluator_id,
+               limit=None,
+               eval_frequency=None,
+               policy_ui={},
```

Comment:
nit.  Setting default method variables to immutable types in python is an anti-pattern.  I would recommend setting this to `None` and then on the first line:

`policy_ui = {} if policy_ui is None else policy_ui`

Reply:
Thanks @hazedav! Always dropping knowledge - I appreciate it!